ocamltest/ocaml_tests.ml

```diff
@@ -96,34 +51,8 @@ let bytecode =
     []
   )
 }
->>>>>>> ocaml/5.1
 
 let native =
-<<<<<<< HEAD
-  let test_actions =
-    if not Ocamltest_config.native_compiler then [skip]
-    else
-      [
-        setup_ocamlopt_opt_build_env;
-        ocamlopt_opt;
-        check_ocamlopt_opt_output;
-        run;
-        check_program_output
-      ]
-  in
-||||||| merged common ancestors
-  let opt_actions =
-  [
-    setup_ocamlopt_byte_build_env;
-    ocamlopt_byte;
-    check_ocamlopt_byte_output;
-    run;
-    check_program_output;
-    setup_ocamlopt_opt_build_env;
-    ocamlopt_opt;
-    check_ocamlopt_opt_output;
-  ] in
-=======
```

Comment:
Same comment above -- should we try to preserve https://github.com/ocaml-flambda/flambda-backend/pull/504 in our resolution?

Reply:
As above